draft-ietf-tls-tls13.md

```diff
@@ -2979,8 +2979,7 @@ Structure of this Message:
           struct {
               opaque configuration_id<1..2^16-1>;
               uint32 expiration_date;
-              NamedGroup group;
-              opaque server_key<1..2^16-1>;
+              KeyShareEntry key_share;
```

Comment:
Please name this something at least a little different from the extension's type name. Something like "static_key_share" would also be a little more descriptive.